docs/patching.rst

```diff
@@ -30,19 +30,17 @@ The Quick and Dirty
 ===================
 
 Very quick, very little explanation. Those with strong git fu may already see
-some shortcuts. Use them! As long as ``mozilla/master`` doesn't have merge
-commits, it's all good.
+some shortcuts. Use them!
 
-Assuming your Github account is ``foobar`` and you've already forked Kitsune::
+First, clone your fork, and then point the master branch to Mozilla's fork.
+Assuming your Github account is ``foobar`` and you've already forked
+Kitsune::
 
-    git clone git@github.com:foobar/kitsune
+    git clone https://github.com/foobar/kitsune
     cd kitsune
     git remote add mozilla https://github.com/mozilla/kitsune.git
     git fetch mozilla
-    git checkout -b temp
-    git branch -d master
-    git checkout -t mozilla/master
-    git branch -d temp
+    git checkout -t mozilla/master -B master
```

Comment:
It also makes your current branch track mozilla/master so if you say 'git pull' in the future, it pulls from mozilla, not origin. It does the same thing as the instructions I deleted, it just does it in one command instead of 4.